### PR TITLE
fix(rhel): drop 51-dracut-rescue-postinst.sh entirely

### PR DESCRIPTION
**Pull request by AdamWill**
It should not be installed in RHEL-8. See #975 and
https://bugzilla.redhat.com/show_bug.cgi?id=1677723 for further
detail; basically this should only ever be installed on RHEL < 8
and Fedora < 31, but it got pinged around a lot and the RHEL-8
branch happened to be forked off at a point where it gets
installed unconditionally. Later changes to make it only install
on RHEL < 8 were committed to master but never to the RHEL-8
branch, so never really did anything. Since this branch is for
RHEL-8 specifically, there's no sense fiddling with conditionals,
we can just ditch it entirely.

Signed-off-by: Adam Williamson <awilliam@redhat.com>

This pull request changes...

## Checklist
- [ ] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

Fixes #

### CODE REVIEW

**Comment by AdamWill** [comment]
This is still waiting for review from the maintainer.

**Comment by lnykryn** [comment]
Could you please file rh bugzilla against rhel8 for this? And for bonus points could you add 
Resolves: #XXXXXXX
line with that bugzilla number to the commit messages?

**Review by haraldh — APPROVED** [review]

**Comment by johannbg** [comment]
@lnykryn can you review this so we can get rid of this PR from the pipeline?

**Pull request closed without merging**
